src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -700,6 +729,13 @@ export class CategoryFacet extends Component {
       args.breadcrumbs.push({ element: categoryFacetBreadcrumbBuilder.build() });
     }
   }
+
+  private isActive() {
```

Comment:
Just to check, isActive will be true when the facet is drilled down and not in it's default state? I had trouble with the name, but I am not able to think of a better name just yet. `isDrilledDown()`, `hasDefaultState` (reversing the logic), `isExpanded`? hmm .... maybe it's good as-is.